includes/documents/class-wcpdf-invoice.php

```diff
@@ -247,6 +247,18 @@ public function init_settings() {
 					'id'				=> 'display_phone',
 				)
 			),
+			array(
+				'type'			=> 'setting',
+				'id'			=> 'display_customer_notes',
+				'title'			=> __( 'Display customer notes', 'woocommerce-pdf-invoices-packing-slips' ),
+				'callback'		=> 'checkbox',
+				'section'		=> 'invoice',
+				'args'			=> array(
+					'option_name'		=> $option_name,
+					'id'				=> 'display_customer_notes',
+					'store_unchecked'	=> true,
+				)
+			),
```

Comment:
The checkbox is now unchecked by default, but it should be enabled for existing customers. Otherwise people update the plugin, they change something in the settings (not noticing the new option, so not changing it either), save the settings and suddenly the notes are no longer displayed.